code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -763,6 +755,16 @@
 /obj/structure/closet/crate/necropolis/bdm/PopulateContents()
 	new /obj/item/melee/transforming/cleaving_saw(src)
 	new /obj/effect/spawner/lootdrop/megafaunaore(src)
+	var/loot = rand(1,4)
+	switch(loot)
```

Comment:
Unneeded variable, just put the rand right in the switch

Reply:
think i did it now, yet not 100% sure